modules/sharedaddy/sharing.css

```diff
@@ -753,3 +753,13 @@ div.sharedaddy.sharedaddy-dark #sharing_email {
 	height: 123px;
 	margin: 0 0 1em 0;
 }
+
+.sd-content .share-customize-link {
+	margin-top: -.7em;
```

Comment:
Let's not use negative margin here, depending on the theme this is going to overlap with the buttons.

<img width="640" alt="Screen Shot 2020-07-17 at 1 56 23 PM" src="https://user-images.githubusercontent.com/1270189/87830210-729dfd80-c835-11ea-96b7-3393917eab05.png">

Reply:
Maybe `margin: 0.1em`? Though, I'll defer to @sixhours or @danhauk here.

<img width="504" alt="Screen Shot 2020-07-17 at 2 01 15 PM" src="https://user-images.githubusercontent.com/1270189/87830500-01127f00-c836-11ea-91f9-ba4cb7e43ad4.png">